src/main/java/svc/controllers/SitemapController.java

```diff
@@ -0,0 +1,51 @@
+package svc.controllers;
+
+import javax.inject.Inject;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import svc.managers.SitemapManager;
+import svc.models.Sitemap;
+
+
+@RestController
+@EnableAutoConfiguration
+public class SitemapController {
+	@Inject
+	SitemapManager sitemapManager;
+	
+	@ResponseBody
+	@RequestMapping(
```

Comment:
Why isn't just `/sitemap`  enough? I don't know why we'd need this separate endpoint with a fileName....?